src/config/typeConfigs.ts

```diff
@@ -284,14 +291,19 @@ export const typeConfigList: Type[] = [
       'Send a WhatsApp message',
     ),
     form: SendWhatsAppMsgForm,
-    // localization: MsgLocalizationForm,
-    // localizeableKeys: ['text', 'quick_replies', 'templating.variables'],
+    localization: WhatsappMsgLocalizationForm,
+    localizeableKeys: [
```

Comment:
What about listOptions titles and descriptions? are they enabled for translations?